### DISCUSSION TITLE
User Authentication

### DISCUSSION THREAD

**Discussion by seanlundberg** [bigcommerce/catalyst#652] [Q&A] [answered]
Hello, we are building a full project on top of Catalyst.

We are currently in the Trial phase at BigCommerce.

When I try to login or create an account, nothing happens, neither on the [demo site](https://catalyst-demo.site/login/) nor on ours.

Is this a placeholder functionality, or does something need to be adjusted on the BigCommerce account in order to enable user authentication and account management?

Cheers!

**Comment by matthewvolk** [comment, marked as answer]
Hi @seanlundberg! 

Super exciting to hear your plans to build a project on top of Catalyst! Definitely keep us posted, we're happy to help however we can 🚀 

Customer registration functionality is currently a placeholder, though we are getting very close. If you take a look at #641, you'll see we're starting to add the first pieces to unlock this functionality — feel free to subscribe to that PR for updates!

For now, if you want to at least try out the logged in experience on the storefront you can [add a new customer in your Control Panel](https://support.bigcommerce.com/s/article/Customer-Account-Management?language=en_US#add-edit) and then sign in as that customer on your Catalyst storefront.

Cheers! 👋

**Reply by seanlundberg** [reply]
Thanks for the quick response @matthewvolk!

Let's say our official launch date is May 1st. Do you envision this functionality being available by then? We can deploy internal dev resources to building it out... But of course, I would rather let the professionals handle it!

**Reply by bookernath** [reply]
@seanlundberg it's likely, but not guaranteed.

**Comment by bookernath** [comment]
@seanlundberg small update - you'll notice some progress on the latest version of the codebase that when you log in you see a customer account area now. We're building out all the pages in that area piece-by-piece now. You'll see a number of PRs over the coming weeks as we make progress on that.
